dashboard/app/models/levels/level.rb

```diff
@@ -29,6 +29,8 @@
 class Level < ApplicationRecord
   include SharedConstants
   include Levels::LevelsWithinLevels
+  include ScriptLevelsHelper
+  include Rails.application.routes.url_helpers
```

Comment:
needed?

Reply:
Apparently so, for this line to work:
https://github.com/code-dot-org/code-dot-org/blob/2a7b5b4a225ce6e04874dde833fefca8f82193bf/dashboard/app/helpers/script_levels_helper.rb#L51